### ISSUE TITLE
Add alt Text to Images in Travelgrid/client/src/components/Custom/CustomCarousel.jsx

### ISSUE THREAD

**Issue by Dippp10** [Adarsh-Chaubey03/TravelGrid#1242]
@Vinayak Jain ji,

Some images in the README.md file lack alt attributes. I would add descriptive alt text to all images to enhance accessibility.

**Comment by Dippp10** [comment]
@Adarsh-Chaubey03 ji i would submit a pull request regarding above issue within a week

**Comment by hasiniraja** [comment]
Please assign this to me ...

**Comment by Dippp10** [comment]
@Adarsh-Chaubey03 ji do not assign to others above issue number 1242 that was created by me.....pull request number 1269 was already merged a week ago....so do close issue number 1242....
